src/test/java/uk/nhs/tis/trainee/actions/service/ActionServiceTest.java

```diff
@@ -286,30 +297,46 @@ void shouldInsertActionWhenPlacementOperationLoad() {
   void shouldNotInsertActionAndDeleteAnyExistingNotCompleteActionsWhenPlacementTypeIgnored() {
     PlacementDto dto = new PlacementDto(TIS_ID, TRAINEE_ID, FUTURE, "ignored placement type");
 
+    when(repository.deleteByTraineeIdAndTisReferenceInfoAndNotComplete(TRAINEE_ID, TIS_ID,
+        String.valueOf(PLACEMENT))).thenReturn(Collections.emptyList());
+
     service.updateActions(Operation.LOAD, dto);
-    verify(repository).deleteByTraineeIdAndTisReferenceInfoAndNotComplete(TRAINEE_ID, TIS_ID,
-        String.valueOf(PLACEMENT));
-    verifyNoMoreInteractions(repository);
+    verify(eventPublishingService, never()).publishActionDeleteEvent(any());
```

Comment:
Should be able to use a more simple `verifyNoInteractions` here since there shouldn't be any other calls to the event publishing service.